### PR TITLE
Update install instructions for caimanmanager

### LINKED ISSUE #227: caimanmanager step of installation instructions is incorrect on macOS
Step 4 of the installation instructions in the README says:

4. Install caimanmanager
```
caimanmanager.py install
```

On my Mac M1 using the default zsh shell, running this returns:
```
zsh: command not found: caimanmanager.py
```

Looking at https://github.com/flatironinstitute/CaImAn#step-2-download-code-samples-and-data-sets, I believe the correct command for all OS should be:
```
caimanmanager install
```

as of [CaImAn v1.9.16](https://github.com/flatironinstitute/CaImAn/releases/tag/v1.9.16)

The later source code blocks should also be updated.

### PR DESCRIPTION
**Pull request by rly**
Fix #227

### CODE REVIEW

**Comment by rly** [comment]
The workflows also use `caimanmanager.py install` after a `git clone` of CaImAn. I'm not sure why that works and running `caimanmanager.py install` after a conda install of CaImAn doesn't.

**Comment by rly** [comment]
Oh CaImAn 1.9.16 was released 5 days ago. The same errors are observed in
https://github.com/nel-lab/mesmerize-core/actions/runs/6428346009/job/17455374529?pr=226
https://github.com/nel-lab/mesmerize-core/actions/runs/6428346005/job/17455374662?pr=226

File: README.md [hunk 1]
```diff
@@ -117,13 +117,13 @@ mamba activate mescore
 4. Install `caimanmanager`
 
 ```bash
-caimanmanager.py install
+caimanmanager install
```
Comment (kushalkolar) [inline README.md:120]: I would add a note above this within step "4." that states this is for caiman v1.9.16 onward, with a link to the old readme for previous versions.

**Review by kushalkolar — CHANGES_REQUESTED** [review]
Thanks! Mostly lgtm, few comments.

File: README.md (hunk 1 above)
Comment (rly) [reply README.md]: Done

**Pull request merged**